src/components/chat/comment-textbox.tsx

```diff
@@ -21,13 +21,19 @@ export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedCommen
                                      { disabled: !commentAdded, "no-action": !commentAdded });
   // resize textarea when user deletes some text
   useEffect(() => {
-    if (textareaRef && textareaRef.current) {
+    if (textareaRef?.current) {
       textareaRef.current.style.height = minTextAreaHeight + "px";
       const scrollHeight = textareaRef.current.scrollHeight;
       textareaRef.current.style.height = scrollHeight + "px";
     }
   });
 
+  const trimContent = (content: string) => {
+    const trimmed = content.trim().replace(/\s+\n/g, "\n");
+    const isEmpty = !trimmed || (trimmed === "\n") || (trimmed === " ");
+    return [trimmed, isEmpty];
```

Comment:
```suggestion
    return [trimmed, isEmpty] as const;
```
You're going to want the `as const` because it eliminates the need to cast below.